src/app/reviewer/[orgSlug]/study/[studyId]/review/study-review-buttons.test.tsx

```diff
@@ -0,0 +1,109 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest'
+import { renderWithProviders, screen, userEvent, mockSessionWithTestData, waitFor } from '@/tests/unit.helpers'
+import { StudyReviewButtons } from './study-review-buttons'
+import { type SelectedStudy, doesTestImageExistForStudyAction } from '@/server/actions/study.actions'
+
+// Mock the action
+vi.mock('@/server/actions/study.actions', async (importOriginal) => {
+    const original = await importOriginal<typeof import('@/server/actions/study.actions')>()
+    return {
+        ...original,
+        doesTestImageExistForStudyAction: vi.fn(),
+    }
+})
+
+// Mock next/navigation
+vi.mock('next/navigation', () => ({
+    useRouter: () => ({
+        push: vi.fn(),
+    }),
+    useParams: () => ({
+        orgSlug: 'test-org',
+    }),
+}))
+
+const mockStudy: SelectedStudy = {
+    id: 'study-123',
+    status: 'PENDING-REVIEW',
+    title: 'Test Study',
+    createdBy: 'Researcher',
+    approvedAt: null,
+    rejectedAt: null,
+    containerLocation: 'loc',
+    createdAt: new Date(),
+    dataSources: [],
+    irbProtocols: null,
+    orgId: 'org-123',
+    outputMimeType: null,
+    piName: 'PI',
+    researcherId: 'res-123',
+    descriptionDocPath: null,
+    irbDocPath: null,
+    reviewerId: null,
+    agreementDocPath: null,
+}
+
+const mockDoesTestImageExistForStudyAction = vi.mocked(doesTestImageExistForStudyAction)
+
+describe('StudyReviewButtons', () => {
+    beforeEach(() => {
+        vi.clearAllMocks()
+    })
```

Comment:
instead of mocking all this, import `mockClerkSession` and `insertTestStudyData` from unit.helpers and use those